torch/jit/_serialization.py

```diff
@@ -146,6 +146,12 @@ def load(f, map_location=None, _extra_files=None):
         import os
         os.remove("scriptmodule.pt")
     """
+
+    # Populate upgrader map in torch.jit.load because
+    # this is the only place we actually use upgraders
+    from torch.jit.operator_upgraders import populate_upgraders_map
```

Comment:
Nit:
Move it to top

Reply:
It creates circular import issue when on the top.

Reply:
Oh? Could you provide more info? If unconditional import causes circular import issue, then that shows our code structure could use some optimization.

Reply:
Ping on this.

Reply:
Ping.

Reply:
With this import `torch.jit.load` now needs `pyyaml` installed. Is this intended?